src/components/Card/Card.tsx

```diff
@@ -11,7 +11,7 @@ type TProps = {
 const Card: React.FC<TProps> = (props: TProps) => {
   return (
     <>
-      <div key={props.movieId} className={styles.movie}>
+      <div key={props.movieId} className={styles.movie} data-testid="movie-1">
```

Comment:
Burdaki data-testid'yi diğerleriyle aynı doğrultuda olması için `Card` olarak set edebiliriz.
+Hiç bi zaman id'lere -1 şeklinde index vermeyelim. Bunları map ediyoruz bazı yerlerde 😄 Her zaman -1 şeklinde kalmıyolar